examples/documentation/PTI_experiment/PTI_Experiment_Recon3D_anisotropic_target_small.py

```diff
@@ -0,0 +1,677 @@
+visual.parallel_4D_viewer(
+    np.transpose(f_tensor, (3, 0, 1, 2)),
+    num_col=4,
+    origin="lower",
+    size=8,
+    titles=[
+        r"$f_{0r}$",
+        r"$f_{0i}$",
+        r"$f_{1c}$",
+        r"$f_{1s}$",
+        r"$f_{2c}$",
+        r"$f_{2s}$",
+        r"$f_{3}$",
+    ],
+    set_title=True,
+)
+
+# %%
+# reconstruct 3D anisotropy (mean permittivity, 3D orientation, optic sign probability)
+# material type:
+# "positive" -> only solution of positively uniaxial material
+# "negative" -> only solution of negatively uniaxial material
+# "unknown" -> both solutions of positively and negatively uniaxial material + optic sign estimation
+
+mean_permittivity, azimuth, theta, mat_map = (
+    setup.scattering_potential_tensor_to_3D_orientation(
+        f_tensor,
+        S_image_tm,
+        material_type="unknown",
+        reg_ret_pr=reg_mean_permittivity,
+        itr=10,
+        fast_gpu_mode=True,
+    )
+)
+
+# %%
+p_mat_map = optics.optic_sign_probability(mat_map, mat_map_thres=0.2)
+phase = optics.phase_inc_correction(
+    f_tensor[0], mean_permittivity[1], theta[1]
+)
+phase_PT, absorption_PT, mean_permittivity_PT = [
+    optics.unit_conversion_from_scattering_potential_to_permittivity(
+        SP_array, lambda_illu, n_media=n_media, imaging_mode="3D"
+    )
+    for SP_array in [phase, f_tensor[1].copy(), mean_permittivity]
+]
+mean_permittivity_PT = np.array(
+    [
+        ((-1) ** i)
+        * util.wavelet_softThreshold(
+            ((-1) ** i) * mean_permittivity_PT[i], "db8", 0.00303, level=1
+        )
+        for i in range(2)
+    ]
+)
+
+
+# %%
+# Visualize reconstructed physical properties of the anisotropic glass target
+
+# %%
+# Reconstructed phase, absorption, mean permittivity, azimuth, and inclination assuming (+) and (-) optic sign
+
+# %%
+# browse the reconstructed physical properties
+visual.parallel_4D_viewer(
+    np.transpose(
+        np.stack(
+            [
+                phase_PT,
+                mean_permittivity_PT[0],
+                azimuth[0],
+                theta[0],
+                absorption_PT,
+                mean_permittivity_PT[1],
+                azimuth[1],
+                theta[1],
+            ]
+        ),
+        (3, 0, 1, 2),
+    ),
+    num_col=4,
+    origin="lower",
+    set_title=True,
+    titles=[
+        r"phase",
+        r"mean permittivity (+)",
+        r"$\omega$ (+)",
+        r"$\theta$ (+)",
+        r"absorption",
+        r"mean permittivity (-)",
+        r"$\omega$ (-)",
+        r"$\theta$ (-)",
+    ],
+)
+
+
+# %%
+# Load the processed results
+PTI_file_name = (
+    "/path/to/Anisotropic_target_small/Anisotropic_target_small_processed.zarr"
+)
+reader = zarr.open(PTI_file_name, mode="r")
+PTI_array = np.array(reader["Row_0/Col_0/f_tensor/array"])
+print(PTI_array.shape)
+PTI_array = np.transpose(PTI_array, (0, 1, 3, 4, 2))[0]
+f_tensor = PTI_array[:7]
+mat_map = PTI_array[7:]
```

Comment:
@talonchandler can you please assign names to the channels, so that we can visualize the processed data by loading in napari? I think the released version of iohub has an API for that.